app/code/Magento/Ui/TemplateEngine/Xhtml/Result.php

```diff
@@ -48,29 +49,36 @@ class Result implements ResultInterface
      * @var JsonHexTag
      */
     private $jsonSerializer;
+    /**
+     * @var State
+     */
+    private $state;
 
     /**
      * @param Template $template
      * @param CompilerInterface $compiler
      * @param UiComponentInterface $component
      * @param Structure $structure
      * @param LoggerInterface $logger
-     * @param JsonHexTag $jsonSerializer
+     * @param JsonHexTag|null $jsonSerializer
+     * @param State|null $state
      */
     public function __construct(
         Template $template,
         CompilerInterface $compiler,
         UiComponentInterface $component,
         Structure $structure,
         LoggerInterface $logger,
-        JsonHexTag $jsonSerializer = null
+        ?JsonHexTag $jsonSerializer = null,
+        ?State $state = null
     ) {
         $this->template = $template;
         $this->compiler = $compiler;
         $this->component = $component;
         $this->structure = $structure;
         $this->logger = $logger;
         $this->jsonSerializer = $jsonSerializer ?? ObjectManager::getInstance()->get(JsonHexTag::class);
+        $this->state = $state ??  ObjectManager::getInstance()->get(State::class);
```

Comment:
Remove the double space before ObjectManager

Reply:
done